### PR TITLE
I have done task OddChecker

### PR DESCRIPTION
**Pull request by Denis-Pikus**

### CODE REVIEW

**Review by integratorBY — APPROVED** [review]
**lgfm**

**Review by drozdi-k — APPROVED** [review]

File: IntensiveJavaGuru/src/main/java/dpdev/sum_of_prime_number/PrimeNumbers.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    public int sumOfPrimeNumber(int num){
+        int sum = 0;
+        int count = 0;
+        for (int i = 2; i <= num; i++){
+            boolean isPrime = true;
+            for (int j = 2; j <= Math.floor(Math.sqrt(i)); j ++) {
+                if ((i % j) == 0){
+                    isPrime = false;
+                    break;
```
Comment (goldenas18) [inline IntensiveJavaGuru/src/main/java/dpdev/sum_of_prime_number/PrimeNumbers.java]: niiiice)

**Review by goldenas18 — APPROVED** [review]
lgfm

**Review by olgaZskiba — COMMENTED** [review]
lgfm

**Pull request merged**